src/maintainer/adding_pkgs.rst

```diff
@@ -761,6 +758,22 @@ For some languages, the community provides tools which can automate this process
 
    The correct and automated packaging of dependency licenses is an ongoing discussion. Please feel free to add your thoughs to `this <https://github.com/conda-forge/conda-forge.github.io/issues/1052>`__ discussion. 
 
+Extra
+-----
+
+Recipe Maintainer
+..................
```

Comment:
I believe this should be a series of ^^^^^^^^ instead.

Reply:
That's right @kathatherine, it should be as you suggested, I also read it [here ](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html?highlight=Headers#sections). But when I changing it from .... to ^^^ it doesn't work, the heading doesn't show up under this section. This seems weird, but I guess (not sure though) it is because we are using this sequence : 
[
===  ---> for Sections
---- ---> for Sub-sections
...... ---> for subsubsections
^^^ ---> for section under subsubsection 
and so on.
]

throughout the file and that might be the reason why this doesn't work.